spec/markdown_spec.js

```diff
@@ -5,9 +5,13 @@ var trim = require("./trim");
 
 describe("markdown_compiler", function () {
 
-  var compiler = require("../markdown");
+  var markdown = require("../markdown");
```

Comment:
I think `markdown_setup` would be a better name here. (similar naming is used elsewhere: https://github.com/brainshave/zetzer/blob/master/spec/dot_spec.js#L4)